meson_options.txt

```diff
@@ -6,6 +6,8 @@ option('rootlibdir', type : 'string',
        description : '''[/usr]/lib/x86_64-linux-gnu or such''')
 option('rootprefix', type : 'string',
        description : '''override the root prefix''')
+option('link-udev-shared', type : 'boolean',
+       description : 'link systemd-udev and its helpers to libsystemd-shared.so')
```

Comment:
/lib/systemd/systemd-udevd, /bin/udevadm and /bin/systemd-hwdb still have a dependency on libsystemd-shared with ```-Dlink-udev-shared=false```